horizons/util/python/decorators.py

```diff
@@ -97,12 +97,11 @@ def __call__(self, *args, **kwargs):
 					del self.cache_dates[key]
 					return self(*args, **kwargs)
 			else:
-				self.cache_dates[key] = time.time() # new entry
+				self.cache_dates[key] = time.time()  # new entry
```

Comment:
Python's official style guide, [PEP 8](http://www.python.org/dev/peps/pep-0008/), is very clear about this issue:
```
Inline comments should be separated by at least two spaces from the statement.
They should start with a # and a single space.
```
UH use PEP 8, but not everything implemented, see point
 -  **String formatting** should prefer the Python3-style format() over the % operator